src/Elm/Parser/State.elm

```diff
@@ -18,9 +18,9 @@ emptyState =
         }
 
 
-storedColumns : State -> List Int
+storedColumns : State -> Maybe Int
 storedColumns (State { indents }) =
-    indents
+    List.head indents
```

Comment:
Let's rename this function, because its name doesn't match its return value anymore.